modules/authorization/proc-determining-permission-policy-and-role-configuration-source.adoc

```diff
@@ -0,0 +1,35 @@
+[id='proc-determining-policy-and-role-source']
+= Determining permission policy and role configuration source
+
+You can configure {product} policy and roles by using different sources.
+To maintain data consistency, {product-short} associates each permission policy and role with one unique source.
+You can only use this source to change the resource.
+
+The available sources are:
+
+Configuration file::
+
+
+Configure roles and policies in the {product-short} `app-config.yaml` configuration file, for instance to xref:enabling-and-giving-access-to-rbac[declare your policy administrators].
++
+The Configuration file pertains to the default `role:default/rbac_admin` role provided by the RBAC plugin.
+The default role has limited permissions to create, read, update, and delete permission policies or roles, and to read catalog entities.
++
+[NOTE]
+====
+In case the default permissions are insufficient for your administrative requirements, you can create a custom admin role with required permission policies.
```

Comment:
```suggestion
In case the default permissions are insufficient for your administrative requirements, you can create a custom admin role with the required permission policies.
```